README.md

```diff
@@ -12,30 +12,30 @@ NOTE! One of the most noticeable shortcoming of the extension is that it allows
 
 ### Magento Newsletter Subscribers
 
-- Add subscribers to Smaily subscribers list
-- Magento built in subscribe newsletter form sends subscribers directly to smaily
-- Magento built-in CAPTCHA and google reCAPTCHA support
+- Add subscribers to Smaily subscribers list;
+- Magento built in subscribe newsletter form sends subscribers directly to smaily;
+- Magento built-in CAPTCHA and google reCAPTCHA support.
 
 ### Magento Products RSS-feed
 
-- Generate default RSS-feed with 50 latest products for easy import to Smaily template
-- Option to customize generated RSS-feed based on products categories
-- Option to limit generated RSS-feed products amount with prefered value
+- Generate default RSS-feed with 50 latest products for easy import to Smaily template;
+- Option to customize generated RSS-feed based on products categories;
+- Option to limit generated RSS-feed products amount with prefered value.
 
 ### Two-way synchronization between Smaily and Magento
 
-- Get unsubscribers from Smaily unsubscribed list
-- Collect new user data for subscribed users
-- Generate data log for each update
+- Get unsubscribers from Smaily unsubscribed list;
+- Collect new user data for subscribed users;
+- Generate data log for each update.
 
 ### Abandoned cart notification
 
-- Automatically notify customers about their abandoned cart
-- Send abandoned cart information to Smaily for easy use on templates
+- Automatically notify customers about their abandoned cart;
+- Send abandoned cart information to Smaily for easy use on templates.
 
 ## Requirements
 
-You'll need to be running Magento 2.0+ for this extension to work. Check specific PHP, Web server, Database etc. requirements for your prefered Magento version from [Magento technology stack requirements](https://devdocs.magento.com/guides/v2.0/install-gde/system-requirements-tech.html)
+You'll need to be running Magento 2.0+ for this extension to work. Check specific PHP, Web server, Database etc. requirements for your prefered Magento version from [Magento technology stack requirements](https://devdocs.magento.com/guides/v2.0/install-gde/system-requirements-tech.html).
```

Comment:
Typos:
- "Web server" should be "web server";
- "Database" should be "database";
- There should be a comma before "etc.";
- "prefered" should be "preferred".